Mastermind.pm

```diff
@@ -209,28 +272,53 @@ sub parse_data {
   my ($hgvs, $gene, $mmcnt1, $mmcnt2, $mmcnt3, $mmid3, $mmuri3) = split /;/, $data; 
```

Comment:
In latest release of Mastermind, the first entry in info field is gene and second is hgvs. The other way around as in current code and the previous Mastermind release. I suggest parsing by `key=value` pairs to handle any other future data releases.

Reply:
Thanks for pointing it out.